packages/elements/src/select/index.ts

```diff
@@ -1074,11 +1035,25 @@ export class Select extends ControlElement implements MultiValue {
    * @return Render template
    */
   protected override render(): TemplateResult {
-    return html` <div id="box">
+    return html`<div
+        id="box"
+        tabindex="0"
+        role="combobox"
+        placeholder=${this.placeholder || nothing}
+        aria-haspopup="listbox"
+        aria-controls="menu"
+        aria-invalid=${this.error ? 'true' : 'false'}
+        aria-expanded=${this.opened ? 'true' : 'false'}
```

Comment:
We can remove short if by do this.
```suggestion
        aria-invalid=${String(this.error)}
        aria-expanded=${String(this.opened)}
```

Reply:
Anything can be input as `error` or `opened` so I think it is better to be explicit that these attributes only accept `'true'` or `'false'`

Reply:
It's OK.